presto-hive-metastore/src/main/java/com/facebook/presto/hive/HiveBucketProperty.java

```diff
@@ -85,6 +101,9 @@ public HiveBucketProperty(
                 storageDescriptor.getNumBuckets(),
                 sortedBy,
                 HIVE_COMPATIBLE,
+                Optional.empty(),
```

Comment:
I think we should introduce clustering related extraction from storage descriptor here as well.

Reply:
Have talked with James before; we may do it later after we can store these information in partition property using thrift structure.